themes/default/content/registry/packages/aws-native/_index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: AWS Native
-meta_desc: Learn how you can use Pulumi's AWS Native Provider to reduce the complexity of provisioning and managing resources on AWS.
+title: AWS Native Cloud Control
+meta_desc: Learn how you can use Pulumi's AWS Native Cloud Control Provider to reduce the complexity of provisioning and managing resources on AWS.
 layout: package
 aliases:
     - "/docs/reference/clouds/aws-native/"
     - "/docs/intro/cloud-providers/aws-native/"
 ---
 
 {{% notes type="info" %}}
-AWS Native is in public preview. AWS Native provides coverage of all resources in the [AWS Cloud Control API](https://aws.amazon.com/blogs/aws/announcing-aws-cloud-control-api/), including same-day access to all new AWS resources. However, some AWS resources are not yet available in AWS Native.
+AWS Native Cloud Control is now Generally Available. AWS Native Cloud Control provides coverage of all resources in the [AWS Cloud Control API](https://aws.amazon.com/blogs/aws/announcing-aws-cloud-control-api/), including same-day access to all new AWS resources. However, some AWS resources are not yet available in AWS Cloud Control.
 
-For new projects, we recommend using AWS Native and [AWS Classic](/registry/packages/aws) side-by-side so you can get the speed and correctness benefits of AWS Native where possible. For existing projects, [AWS Classic](/registry/packages/aws) remains fully supported; at this time, we recommend waiting to migrate existing projects to AWS Native.
+For new projects, we recommend using AWS Native and [AWS Classic](/registry/packages/aws) side-by-side so you can get the speed and correctness benefits of AWS Native where possible. For existing projects, [AWS Classic](/registry/packages/aws) remains fully supported; we recommend waiting to fully migrate existing projects to AWS Native Cloud Control.
```

Comment:
This language still kind of presents AWS Native CC as an AWS provider alternative, rather than a complement. maybe "The AWS Native Cloud Control provider serves as a complement to resources in the AWS classic provider" We recommend both new and existing projects consider adopting AWS Native Cloud Control for  resources that may be unavailable in Classic or where you need quick access to the latest features."